.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,16 @@
+name: GitHub Actions Demo
+on: [push]
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ls -l
+      - run: docker build --target test --tag todo-app:test .
+      - name: Docker Run
```

Comment:
You could make this a bit more descriptive by calling it "Run tests in Docker" or something.